### PR TITLE
Replace getattr with hasattr to avoid errors

### PR DESCRIPTION
**Pull request by mumarkhan999**

### CODE REVIEW

**Review by danialmalik — APPROVED** [review]

**Review by HamzaIbnFarooq — APPROVED** [review]
We should have a feature flag instead of that but as we have come to the conclusion that this piece of code will be removed in near future. I am approving this.

File: openedx_caliper_tracking/processor.py [hunk 1]
```diff
@@ -111,7 +111,9 @@ def __call__(self, event):
 
             CALIPER_LOGGER.info(json.dumps(transformed_event))
 
-            if getattr(settings, 'CALIPER_DELIVERY_ENDPOINT') and getattr(settings, 'CALIPER_DELIVERY_AUTH_TOKEN'):
+            if (settings.FEATURES.get('ENABLE_CALIPER_DELIVERY_POINT')
```
Comment (danialmalik) [inline openedx_caliper_tracking/processor.py]:
```suggestion
            if (settings.FEATURES.get('ENABLE_CALIPER_EVENTS_DELIVERY')
```

**Review by tasawernawaz — COMMENTED** [review]
I would recommend that moving farword we should add tests for our changes.

File: openedx_caliper_tracking/processor.py (hunk 1 above)
Comment (mumarkhan999) [reply openedx_caliper_tracking/processor.py]: done in the latest commit.

**Comment by mumarkhan999** [comment]
> We should have a feature flag instead of that but as we have come to the conclusion that this piece of code will be removed in near future. I am approving this.

added.

**Comment by mumarkhan999** [comment]
> I would recommend that moving farword we should add tests for our changes.

tests would be added in another PR.

**Pull request merged**